README.md

```diff
@@ -122,20 +122,20 @@ static String eWONCertificatePath = "/usr/Certificates";
 ```
 ### ServerConfig
 
-The OSIsoft PI Database Web API must be installed for this connector to work.  The connector requires three items to connect to the OSIsoft PI Web API they are the IP address of the server running the PI Web API, WebID of the PI Web API and a valid user name and password for the system running the PI Web API software.
+The OSIsoft PI Database Web API must be installed for this connector to work.  The connector requires three items to connect to the OSIsoft PI Web API they are the URL of the server running the PI Web API, WebID of the PI Web API and a valid user name and password for the system running the PI Web API software.
 
-IP – IP address of the system running the Web API (in this example assume the IP address is 192.168.0.124)
+URL – IP address or domain name of the system running the Web API (in this example assume the IP address is 192.168.0.124)
 
-WebID – WebID of your OSIsoft Web API.  This value can be attained by pointing a browser to the IP address (https://192.168.0.124/piwebapi/dataservers).  The WebID is one of several parameters returned.  A username and password will be required to access the page.  If a username is not required it may be cached.  Try clearing the cache to verify the username and password.  The same username and password will be used to create the credentials in the next step.  (A security warning may occur when trying to access this page, this warning should be ignored)
+WebID – WebID of your OSIsoft Web API.  This value can be attained by pointing a browser to the OSIsoft PIWEBAPI URL (https://192.168.0.124/piwebapi/dataservers).  The WebID is one of several parameters returned.  A username and password will be required to access the page.  If a username is not required it may be cached.  Try clearing the cache to verify the username and password.  The same username and password will be used to create the credentials in the next step.  (A security warning may occur when trying to access this page, this warning should be ignored)
 
 #### Proxy URL
 
-Adding the field "ProxyURL" under the server config section will change request endpoints to https://server-ip/proxy-url. This field is optional. 
+Adding the field "ProxyURL" under the server config section will change request endpoints to https://server-url/proxy-url. This field is optional. 
```

Comment:
Shouldnt it be proxy path rather than proxy url?

Reply:
proxy url is being removed in the next PR so I did not make any large changes to it here. Instead of a proxy URL parameter users will need to specify the full path and know which path to use for their desired API.